src/control/server/storage/scm/provider.go

```diff
@@ -641,6 +653,8 @@ func (p *Provider) formatRamdisk(req FormatRequest) (*FormatResponse, error) {
 	return &FormatResponse{
 		Mountpoint: res.Target,
 		Formatted:  res.Mounted,
```

Comment:
Is Formatted: res.Mounted correct?  I am trying to gain more context, but would have thought that res.Formatted would make more sense.

Reply:
There is no Formatted member of a MountResponse struct. In the case of a ramdisk, there isn't really a format step, but I figured I'd use Mounted-ness as an extra check. For clarity's sake though I think I'll just hard-code it to true since we can't get to this result unless the mount succeeded anyhow.